test/SVMSpokePoolClient.fills.ts

```diff
@@ -201,4 +203,36 @@ describe("SVMSpokePoolClient: Fills", function () {
     );
     expect(fillStatusAfterRequest).to.equal(FillStatus.RequestedSlowFill);
   });
+
+  it("Closes the fill pda after the fill deadline has passed", async () => {
+    const newRelayData = { ...relayData, depositId: new Uint8Array(intToU8Array32(getRandomInt())) };
+    const formattedRelayData = formatRelayData(newRelayData);
+    await mintTokens(signer, solanaClient, mint.address, BigInt(relayData.outputAmount));
+    const { fillInput, relayData: fillRelayData } = await sendCreateFill(
+      solanaClient,
+      signer,
+      mint,
+      decimals,
+      newRelayData
+    );
+
+    const fillStatusAfterFill = await spokePoolClient.relayFillStatus(formattedRelayData);
+    expect(fillStatusAfterFill).to.equal(FillStatus.Filled);
+
+    try {
+      await closeFillPda(signer, solanaClient, fillInput.fillStatus);
+    } catch (error) {
+      expect(error.context.logs.some((log) => log.includes("The fill deadline has not passed!"))).to.be.true;
+    }
+
+    await setCurrentTime(signer, solanaClient, fillRelayData.fillDeadline + 1);
+
+    await closeFillPda(signer, solanaClient, fillInput.fillStatus);
+
+    const fillStatusAccount = await fetchEncodedAccount(solanaClient.rpc, fillInput.fillStatus, {
+      commitment: "confirmed",
+    });
+
+    expect(fillStatusAccount.exists).to.be.false;
```

Comment:
Could we add another check here calling relayFillStatus at the end to be sure that it continues to return Filled even if the fillStatusPDA is closed?
Something like:
```suggestion
    expect(fillStatusAccount.exists).to.be.false;
    
    const fillStatusWithPdaClosed = await spokePoolClient.relayFillStatus(formattedRelayData);
    expect(fillStatusWithPdaClosed).to.equal(FillStatus.Filled);
```